tools/create-translation-issues.sh

```diff
@@ -0,0 +1,216 @@
+#!/bin/bash
```

Comment:
Great work! I assume this will make the process a lot easier. Since there are many other release scripts too, shouldn't we put them in folder called /release under /tools/?

Reply:
`tools` used to be about other repositories contributing to Jamulus.

Reply:
> Since there are many other release scripts too, shouldn't we put them in folder called /release under /tools/?

I was rather wondering if we should move such things to a separate repo as there is no need to distribute them as part of the releases. Sure, it's handy for us to have them available right next to the code, but moving them outside may also have benefits.

Not sure what the scope of the repo and as such what the name should be. Should it be limited to release `tools`? Or general `automation` maybe?

Reply:
Probably automation or maintain? Not sure if tools is correct

Reply:
I suggest merging this PR with the create-translation-issues.sh script and move it afterwards along with the other scripts. I've opened #1764 to track this.

Reply:
That, in itself, has benefits, as it implies it's more of an overview - covering all assets, not either jamulussoftware/jamulus or jamulussoftware/jamuluswebsite as would currently be the case.  Seems a positive step to me.